src/Controller/DatatransResponseController.php

```diff
@@ -95,8 +106,16 @@ public function processSuccessResponse(Request $request, PaymentInterface $payme
    *
    * @throws \Exception
    */
-  public function processErrorResponse(PaymentInterface $payment) {
-
+  public function processErrorResponse(Request $request, PaymentInterface $payment) {
+    if (\Drupal::state()->get('datatrans.debug', FALSE)) {
```

Comment:
In case we actually want to offer this as a setting somewhere, it might be more useful to deal with this with configuration, not state. We could even make it part of the payment method settings.